### PR TITLE
asDeploymentHash sanitizer

### PR DESCRIPTION
**Pull request by rzadp**
Addresses https://github.com/UniversalLogin/UniversalLoginSDK/pull/1299#discussion_r334948026

### CODE REVIEW

File: universal-login-commons/lib/core/utils/sanitizers/asProperHash.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import {utils} from 'ethers';
+import {Sanitizer, Result} from '@restless/sanitizers';
+
+export const asProperHash : (length: number) => Sanitizer<string> = (length: number) => {
```
Comment (Marik-D) [inline universal-login-commons/lib/core/utils/sanitizers/asProperHash.ts]: rename to `asHexString`

Comment (Marik-D) [reply universal-login-commons/lib/core/utils/sanitizers/asProperHash.ts]: Also pl remove the spaces before `:`

Comment (rzadp) [reply universal-login-commons/lib/core/utils/sanitizers/asProperHash.ts]: Done

File: universal-login-commons/lib/core/utils/sanitizers/asHexString.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import {utils} from 'ethers';
+import {Sanitizer, Result} from '@restless/sanitizers';
+
+export const asHexString: (length: number) => Sanitizer<string> = (length: number) => {
+  return (value: unknown, path: string) => {
+    if (typeof value === 'string' && utils.isHexString(value) && value.length === length) {
```
Comment (xitronix) [inline universal-login-commons/lib/core/utils/sanitizers/asHexString.ts]: Make consistent with `isProperHexString(..)`
https://github.com/UniversalLogin/UniversalLoginSDK/blob/master/universal-login-commons/lib/core/utils/hexStrings.ts

Comment (Marik-D) [reply universal-login-commons/lib/core/utils/sanitizers/asHexString.ts]: We should delete our own `isProperHexString` implementation and use the one in etheres

**Pull request merged**